tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc

```diff
@@ -111,14 +111,21 @@ else
   $(error "TOOLCHAIN=$(TOOLCHAIN) is not supported.")
 endif
 
+# TODO: fix CMSIS-NN related warnings.
```

Comment:
please add a github issue to the TODO.

e.g. TODO(#1234):

Reply:
Done